app/views/proposal/_tooltip.html.haml

```diff
@@ -3,33 +3,21 @@
   %li{'class'=>class_for_todo(progress_status['registered'])}
     %span{'class'=>icon_for_todo(progress_status['registered'])}
       - if progress_status['registered']
-        = link_to 'Edit your registration', edit_conference_conference_registrations_path(event.program.conference.short_title)
+        = link_to 'Register to the conference', edit_conference_conference_registrations_path(event.program.conference.short_title)
```

Comment:
The TODO list content should not change. The TODO is always 'Register to the conference' no matter if it's done or not.